.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,35 @@
+name: PR
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  docker-push:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set Tag and SHA
+        run: |
+          CLEAN_TAG=$(echo "${{ github.event.pull_request.head.ref  }}"  | tr / -)
+          echo "TAG=$CLEAN_TAG" >> $GITHUB_ENV
+          echo "SHA=${{ github.event.pull_request.head.sha }}" >> $GITHUB_ENV
+      - name: Build
+        run: >
+          docker build -t onsdigital/eq-runner-maintenance-page:$TAG
+          -t ${{ secrets.GAR_LOCATION }}/${{ secrets.GAR_PROJECT_ID }}/docker-images/eq-runner-maintenance-page:$TAG .
+      - name: Push to Docker Hub
```

Comment:
I think if we want to keep pushing to Docker hub, then this should also happen on merges to Master to be consistent.

Reply:
On balance, and taking @parimooa comment into consideration, I'm not sure there is much point in pushing to Dockerhub at all. Unlike Runner we won't have a docker-compose for this image (for a quickstart for example) and therefore have less need for it to be available in Dockerhub. I will remove this step.

Reply:
Removed Dockerhub push